### ISSUE TITLE
Tab stops in main navigation menu are incorrect

### ISSUE THREAD

**Issue by dananjohnson** [ManifoldScholar/manifold#2187]
`tabindex` values need to be adjusted to prevent each link from having a tab stop.

See https://www.w3.org/TR/wai-aria-practices-1.1/examples/menubar/menubar-1/menubar-1.html#sc_label for reference.

**Comment by dananjohnson** [comment]
The current desktop nav menu implementation seems correct, as it doesn't fit the WAI-ARIA definition of a menu:
> A menu is usually opened, or made visible, by activating a menu button, choosing an item in a menu that opens a sub menu

The mobile nav menu may benefit from being treated as a [`menubar`](https://www.w3.org/TR/wai-aria-practices-1.1/examples/menubar/menubar-1/menubar-1.html). However, we'll need more guidance on this. Besides updating `tabindex` values for menu links, we'll need to set up key event listeners to allow navigation and opening/closing of submenus using arrow keys. Further, while it's true that each link currently has a tab stop, I'm not sure that this a problem, given that these stops will be ignored until the user explicitly expands the nav menu, and enforcing keyboard navigation within the menu would not allow any fewer keystrokes than navigating with `tab`/`shift + tab`.

**Comment by dananjohnson** [comment]
Some smaller accessibility improvements to the nav menus have been made in #2203.

**Comment by avaaec** [comment]
Hi! Ava from the UW ATCAX team here, I would love to help but I'm not sure if I understand which menu this is talking about. Could you explain where the "main navigation menu" in question is? Additionally if this is in response to one of our filled issues, knowing exactly which one would be very helpful.

**Comment by dananjohnson** [comment]
Hi @avaaec! This is in response to issue 4 filled by your team:
> Implementation of navigation menu in banner is incorrect, has a tabstop for each link

I wasn't sure if you were referring to the desktop or mobile navigation menu in the banner, so I looked into both, but still couldn't see the issue in question.

For reference, I'm referring to these:
<img width="1233" alt="Screen Shot 2019-06-24 at 9 24 01 AM" src="https://user-images.githubusercontent.com/12454203/60036020-1634d280-9663-11e9-8910-68adf9d70f27.png">

<img width="439" alt="Screen Shot 2019-06-24 at 9 25 11 AM" src="https://user-images.githubusercontent.com/12454203/60036047-2056d100-9663-11e9-97df-7311569350a5.png">
<img width="421" alt="Screen Shot 2019-06-24 at 9 25 02 AM" src="https://user-images.githubusercontent.com/12454203/60036059-25b41b80-9663-11e9-8559-6a66b666d9c7.png">

**Comment by zdavis** [comment]
@avaaec — just want to make sure that you saw our feedback on this issue!

**Comment by avaaec** [comment]
Hi, sorry I neglected this issue for so long, our workload has been pretty crazy.

I have spoken with the person who originally reported this bug and seems we may have miscommunicated around the issue. The current use of links in the home page banner is fine. If the page content was changed dynamically on the page it would be expected that these links were a tab control but the links do not dynamically change the page content. They are...links. That take you to different pages. Not sure how this was misunderstood when we filled this issue. 

Sorry if we wasted your time with this one, I think we can close it for good.
